cognite/neat/graph/queries/_base.py

```diff
@@ -98,55 +98,48 @@ def describe(
         self,
         instance_id: URIRef,
         property_renaming_config: dict | None = None,
-    ) -> tuple[str, dict[str, list[str]]]:
+    ) -> tuple[str, dict[str, list[str]]] | None:
         """DESCRIBE instance for a given class from the graph store
 
         Args:
             instance_id: Instance id for which we want to generate query
-            property_rename_config: Dictionary to rename properties, default None
+            property_renaming_config: Dictionary to rename properties, default None
 
         Returns:
             Dictionary of instance properties
         """
-
         property_values: dict[str, list[str]] = defaultdict(list)
-
-        for subject, predicate, object_ in cast(list[ResultRow], self.graph.query(f"DESCRIBE <{instance_id}>")):
-            if object_.lower() not in [
+        identifier = remove_namespace_from_uri(instance_id, validation="prefix")
```

Comment:
The subject will be the same for each iteration, so only calling `remove_namespace_uri` once on it save some time